tests/baseline/tsfresh_features_test.py

```diff
@@ -0,0 +1,417 @@
+
+            def getKey(item):
+                return item[0]
+
+            sorted_feature_names = sorted(feature_names, key=getKey)
+            sorted_tsfresh_features = sorted(TSFRESH_FEATURES, key=getKey)
+
+            for nid, nname in sorted_feature_names:
+                new_entry = None
+                if int(nid) > max_known_id:
+                    new_entry = '    [%s, %s]' % (str(nnid), str(nname))
+                    fail_msg = '''
+
+###    NOTICE    ###
+
+If you adding/testing new features, you can create a new baseline from: %s
+
+Added the new baseline (Python 2 and 3 baselines ARE different) as either:
+Local path: %s/tsfresh-<NEW_VERSION>.py{2,3}.data.json.features.transposed.csv
+Repo path: tests/baseline/tsfresh-<NEW_VERSION>.py{2,3}.data.json.features.transposed.csv
+
+In the tsfresh_feature_names.py change the following:
+Local path: %s/tsfresh_feature_names.py
+Repo path: tests/baseline/tsfresh_feature_names.py
+
+- TSFRESH_BASELINE_VERSION to your <NEW_VERSION>
+- Add the new feature/s name to TSFRESH_FEATURES with a incremented id value
+
+New entry for tsfresh_feature_names.py:
+%s
+''' % (t_fname_out_fail, baseline_dir, baseline_dir, str(new_entry))
+                    if new_entry:
+                        shutil.move(t_fname_out, t_fname_out_fail)
+                    self.assertEqual(new_entry, None, msg=fail_msg)
+
+                for oid, oname in sorted_feature_names:
+                    if int(oid) == int(nid):
+                        if str(oname) != str(nname):
+                            soid = str(int(oid))
+                            snid = str(int(nid))
+                            soname = str(oname)
+                            snname = str(oname)
+                            fail_msg = '''
+
+###    ERROR    ###
+
+A baseline feature name for id %s has changed:
+    [%s, %s]
+Calculated feature name for id %s was:
+    [%s, %s]
+
+If you adding/testing new features, you can create a new baseline from: %s
+
+Added the new baseline (Python 2 and 3 baselines ARE different) as either:
+Local path: %s/tsfresh-<NEW_VERSION>.py{2,3}.data.json.features.transposed.csv
+Repo path: tests/baseline/tsfresh-<NEW_VERSION>.py{2,3}.data.json.features.transposed.csv
+
+In the tsfresh_feature_names.py change the following:
+Local path: %s/tsfresh_feature_names.py
+Repo path: tests/baseline/tsfresh_feature_names.py
+
+- TSFRESH_BASELINE_VERSION to your <NEW_VERSION>
+- Add the new feature/s name to TSFRESH_FEATURES with a incremented id value
+
+''' % (soid, soid, soname, soid, soid, snname, t_fname_out, baseline_dir, baseline_dir)
+                            if soname != snname:
+                                shutil.move(t_fname_out, t_fname_out_fail)
+                            self.assertEqual(soname, snname, msg=fail_msg)
+            feature_names_match = True
+        else:
+            feature_names_match = False
+            shutil.move(t_fname_out, t_fname_out_fail)
```

Comment:
@nils-braun Yes it could be in the docs, but look those blow up message strings give the user exact info about where and what on their own environment, docs cannot do that.

```python
>       self.assertTrue(features_equal, msg=fail_msg)
E       AssertionError:
E
E       ###    NOTICE    ###
E
E       If you adding/testing new features, you can create a new baseline from: /opt/python_virtualenv/projects/tsfresh-py2712/apps/tsfresh_test_tsfresh/tsfresh/tsfresh/examples/data/test_tsfresh_baseline_dataset/tsfresh-unknown-version.py2.data.json.features.transposed.csv
E
E       Added the new baseline (Python 2 and 3 baselines ARE different) as either:
E       Local path: /opt/python_virtualenv/projects/tsfresh-py2712/apps/tsfresh_test_tsfresh/tsfresh/tests/baseline/tsfresh-<NEW_VERSION>.py{2,3}.data.json.features.transposed.csv
E       Repo path: tests/baseline/tsfresh-<NEW_VERSION>.py{2,3}.data.json.features.transposed.csv
E
E       In the tsfresh_feature_names.py change the following:
E       Local path: /opt/python_virtualenv/projects/tsfresh-py2712/apps/tsfresh_test_tsfresh/tsfresh/tests/baseline/tsfresh_feature_names.py
E       Repo path: tests/baseline/tsfresh_feature_names.py
E
E       - TSFRESH_BASELINE_VERSION to your <NEW_VERSION>
E       - Add the new feature/s name to TSFRESH_FEATURES with a incremented id value
E
E       NOT in baseline   :: [['value__variance', '1615724.45417'], ['value__time_reversal_asymmetry_statistic__lag_3', '-35027911339.3'], ['value__time_reversal_asymmetry_statistic__lag_2', '-34205999048.4'], ['value__time_reversal_asymmetry_statistic__lag_1', '-32737222382.9'], ['value__sum_values', '609165.0'], ['value__standard_deviation', '1271.11150344'], ['value__skewness', '7.72297209044'], ['value__sample_entropy', '0.0350711424523'], ['value__quantile__q_0.9', '10057.6'], ['value__quantile__q_0.8', '10034.6'], ['value__quantile__q_0.7', '10019.3'], ['value__quantile__q_0.6', '10008.4'], ['value__quantile__q_0.4', '9976.6'], ['value__quantile__q_0.3', '9964.4'], ['value__quantile__q_0.2', '9945.2'], ['value__quantile__q_0.1', '9920.8'], ['value__number_peaks__n_3', '8.0'], ['value__number_cwt_peaks__n_5', '7.0'], ['value__median', '9987.0'], ['value__mean_second_derivate_central', '86.9137931034'], ['value__mean_change', '-167.101694915'], ['value__mean_autocorrelation', '-0.0386679248489'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.8', '66.5'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.6', '43.3571428571'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.4', '45.12'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.0', '216.186440678'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.6', '19.0'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.4', '22.4285714286'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.2', '31.0454545455'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.0', '42.2051282051'], ['value__mean_abs_change_quantiles__qh_0.6__ql_0.4', '8.0'], ['value__mean_abs_change_quantiles__qh_0.6__ql_0.0', '41.9259259259'], ['value__mean_abs_change_quantiles__qh_0.4__ql_0.0', '34.5'], ['value__mean_abs_change_quantiles__qh_0.2__ql_0.0', '32.1666666667'], ['value__mean_abs_change', '216.186440678'], ['value__mean', '10152.75'], ['value__maximum', '19907.0'], ['value__longest_strike_below_mean', '59.0'], ['value__longest_strike_above_mean', '1.0'], ['value__last_location_of_maximum', '0.0166666666667'], ['value__large_standard_deviation__r_0.2', '0.0'], ['value__large_standard_deviation__r_0.15', '0.0'], ['value__kurtosis', '59.7588328709'], ['value__index_mass_quantile__q_0.9', '0.9'], ['value__index_mass_quantile__q_0.8', '0.8'], ['value__index_mass_quantile__q_0.7', '0.7'], ['value__index_mass_quantile__q_0.6', '0.6'], ['value__index_mass_quantile__q_0.4', '0.4'], ['value__index_mass_quantile__q_0.3', '0.3'], ['value__index_mass_quantile__q_0.2', '0.2'], ['value__index_mass_quantile__q_0.1', '0.1'], ['value__first_location_of_maximum', '0.0'], ['value__fft_coefficient__coeff_9', '9752.0'], ['value__fft_coefficient__coeff_8', '10118.1878223'], ['value__fft_coefficient__coeff_7', '10112.2353065'], ['value__fft_coefficient__coeff_6', '10013.0'], ['value__fft_coefficient__coeff_5', '9941.06920387'], ['value__fft_coefficient__coeff_4', '10122.4450394'], ['value__fft_coefficient__coeff_3', '9662.0'], ['value__fft_coefficient__coeff_2', '9946.86713832'], ['value__fft_coefficient__coeff_1', '9999.19548961'], ['value__fft_coefficient__coeff_0', '188744.0'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_5', '5098.00211096'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_20', '35889.4097023'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_2', '37.7423968424'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_10', '17699.4186214'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_5', '6971.3981125'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_20', '34539.322379'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_2', '-52.8488952281'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_10', '17545.3115711'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_5', '8961.24545201'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_20', '33073.8398118'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_2', '-258.360817559'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_10', '17020.021077'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_5', '10809.1403497'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_20', '31514.2745692'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_2', '-580.770680481'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_10', '16110.629157'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_5', '12187.7002319'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_20', '29860.2355567'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_2', '-686.478984504'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_10', '14812.096725'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_5', '12756.2643883'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_20', '28126.6226665'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_2', '572.862909845'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_10', '13140.2738663'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_5', '12237.4005411'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_20', '26328.0339575'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_2', '4437.77480107'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_10', '11126.916712'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_5', '10495.3490812'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_20', '24477.4931576'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_2', '9560.37075666'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_10', '8818.83329028'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_5', '7592.18099983'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_20', '22583.2151358'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_2', '11058.4516021'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_10', '6275.52786852'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_14__w_5', '382.270435327'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_14__w_20', '40759.7307605'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_14__w_10', '14247.9900889'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_13__w_5', '757.384327655'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_13__w_20', '40049.1288683'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_13__w_10', '15339.8489225'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_12__w_5', '1360.48823936'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_12__w_20', '39198.5723816'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_12__w_10', '16281.1554379'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_11__w_5', '2259.71943991'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_11__w_20', '38216.8083894'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_11__w_10', '17021.3079358'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_5', '3504.27096948'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_20', '37112.5273411'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_2', '34.237526701'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_10', '17510.0683865'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_5', '3800.59796118'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_20', '20656.5373632'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_2', '5532.44457496'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_10', '3569.09017076'], ['value__count_below_mean', '59.0'], ['value__count_above_mean', '1.0'], ['value__binned_entropy__max_bins_10', '0.0847660757148'], ['value__autocorrelation__lag_9', '0.00814393431441'], ['value__autocorrelation__lag_8', '-0.224079469613'], ['value__autocorrelation__lag_7', '-0.128465776323'], ['value__autocorrelation__lag_6', '-0.293723534459'], ['value__autocorrelation__lag_5', '-0.139640891504'], ['value__autocorrelation__lag_4', '-0.146344370653'], ['value__autocorrelation__lag_3', '-0.0258892339729'], ['value__autocorrelation__lag_2', '-0.0673812431381'], ['value__autocorrelation__lag_1', '-0.287286927404'], ['value__augmented_dickey_fuller', '-1.03295363406'], ['value__ar_coefficient__k_10__coeff_4', '0.293112171115'], ['value__ar_coefficient__k_10__coeff_3', '0.0291310010511'], ['value__ar_coefficient__k_10__coeff_2', '-0.070336909309'], ['value__ar_coefficient__k_10__coeff_1', '0.149377184681'], ['value__ar_coefficient__k_10__coeff_0', '1255.63187635'], ['value__approximate_entropy__m_2__r_0.9', '0.00118402185711'], ['value__approximate_entropy__m_2__r_0.7', '0.00118402185711'], ['value__approximate_entropy__m_2__r_0.5', '0.00118402185711'], ['value__approximate_entropy__m_2__r_0.3', '0.00118402185711'], ['value__approximate_entropy__m_2__r_0.1', '0.0992138207035'], ['value__absolute_sum_of_changes', '12755.0'], ['value__abs_energy', '6281643421.0']]
E
E       NOT in calculated :: [['value__variance', '3196.67638889'], ['value__time_reversal_asymmetry_statistic__lag_3', '1993963105.11'], ['value__time_reversal_asymmetry_statistic__lag_2', '1215832201.59'], ['value__time_reversal_asymmetry_statistic__lag_1', '836027272.276'], ['value__sum_values', '599165.0'], ['value__standard_deviation', '56.5391580136'], ['value__skewness', '-0.135422921254'], ['value__sample_entropy', '2.22646139752'], ['value__quantile__q_0.9', '10048.9'], ['value__quantile__q_0.8', '10033.2'], ['value__quantile__q_0.7', '10017.6'], ['value__quantile__q_0.6', '10004.8'], ['value__quantile__q_0.4', '9975.6'], ['value__quantile__q_0.3', '9962.7'], ['value__quantile__q_0.2', '9937.8'], ['value__quantile__q_0.1', '9909.9'], ['value__number_peaks__n_3', '9.0'], ['value__number_cwt_peaks__n_5', '6.0'], ['value__median', '9981.0'], ['value__mean_second_derivate_central', '0.706896551724'], ['value__mean_change', '2.38983050847'], ['value__mean_autocorrelation', '1.1720475294'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.8', '58.3333333333'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.6', '41.125'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.4', '45.0740740741'], ['value__mean_abs_change_quantiles__qh_1.0__ql_0.0', '46.6949152542'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.6', '13.0'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.4', '20.0'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.2', '31.619047619'], ['value__mean_abs_change_quantiles__qh_0.8__ql_0.0', '43.0256410256'], ['value__mean_abs_change_quantiles__qh_0.6__ql_0.4', '6.0'], ['value__mean_abs_change_quantiles__qh_0.6__ql_0.0', '42.25'], ['value__mean_abs_change_quantiles__qh_0.4__ql_0.0', '35.6'], ['value__mean_abs_change_quantiles__qh_0.2__ql_0.0', '29.4'], ['value__mean_abs_change', '46.6949152542'], ['value__mean', '9986.08333333'], ['value__maximum', '10123.0'], ['value__longest_strike_below_mean', '9.0'], ['value__longest_strike_above_mean', '10.0'], ['value__last_location_of_maximum', '0.966666666667'], ['value__large_standard_deviation__r_0.2', '1.0'], ['value__large_standard_deviation__r_0.15', '1.0'], ['value__kurtosis', '-0.0289015217515'], ['value__index_mass_quantile__q_0.9', '0.916666666667'], ['value__index_mass_quantile__q_0.8', '0.816666666667'], ['value__index_mass_quantile__q_0.7', '0.716666666667'], ['value__index_mass_quantile__q_0.6', '0.616666666667'], ['value__index_mass_quantile__q_0.4', '0.416666666667'], ['value__index_mass_quantile__q_0.3', '0.316666666667'], ['value__index_mass_quantile__q_0.2', '0.216666666667'], ['value__index_mass_quantile__q_0.1', '0.116666666667'], ['value__first_location_of_maximum', '0.95'], ['value__fft_coefficient__coeff_9', '-248.0'], ['value__fft_coefficient__coeff_8', '118.187822328'], ['value__fft_coefficient__coeff_7', '112.235306522'], ['value__fft_coefficient__coeff_6', '13.0'], ['value__fft_coefficient__coeff_5', '-58.9307961342'], ['value__fft_coefficient__coeff_4', '122.445039355'], ['value__fft_coefficient__coeff_3', '-338.0'], ['value__fft_coefficient__coeff_2', '-53.1328616833'], ['value__fft_coefficient__coeff_1', '-0.804510387479'], ['value__fft_coefficient__coeff_0', '178744.0'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_5', '6826.23862162'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_20', '34437.5408409'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_2', '50.2590300879'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_9__w_10', '17169.076641'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_5', '8545.47382822'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_20', '32985.8151195'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_2', '17.9554856918'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_8__w_10', '16639.5484491'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_5', '10110.8994435'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_20', '31428.5198149'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_2', '39.9167672586'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_7__w_10', '15737.7224437'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_5', '11253.943681'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_20', '29788.0657403'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_2', '336.632431542'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_6__w_10', '14466.1094898'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_5', '11696.157551'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_20', '28065.040621'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_2', '1295.74336719'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_5__w_10', '12835.3989402'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_5', '11207.9294065'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_20', '26275.1871817'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_2', '3308.43040143'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_4__w_10', '10876.5237364'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_5', '9670.13169234'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_20', '24433.795925'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_2', '6017.19200793'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_3__w_10', '8634.72484753'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_5', '7120.96092089'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_20', '22554.4207082'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_2', '7535.02284446'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_2__w_10', '6167.81421141'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_5', '3771.32544152'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_20', '20645.6350314'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_2', '5485.74118013'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_1__w_10', '3543.0796763'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_14__w_5', '1019.57078515'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_14__w_20', '39919.0572501'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_14__w_10', '15154.9058723'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_13__w_5', '1652.10185551'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_13__w_20', '39076.9898057'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_13__w_10', '16046.1857704'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_12__w_5', '2542.01968757'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_12__w_20', '38098.1939127'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_12__w_10', '16737.7538459'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_11__w_5', '3717.9693031'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_11__w_20', '36992.8147885'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_11__w_10', '17183.302683'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_5', '5169.35388762'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_20', '35770.923232'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_2', '35.9047024745'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_10__w_10', '17339.7697657'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_5', '-20.2575971343'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_20', '18718.9572589'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_2', '-40.2658469608'], ['value__cwt_coefficients__widths_(2, 5, 10, 20)__coeff_0__w_10', '836.64197854'], ['value__count_below_mean', '31.0'], ['value__count_above_mean', '29.0'], ['value__binned_entropy__max_bins_10', '2.11974362674'], ['value__autocorrelation__lag_9', '0.217484000968'], ['value__autocorrelation__lag_8', '0.360082254297'], ['value__autocorrelation__lag_7', '0.653853495147'], ['value__autocorrelation__lag_6', '0.512480168514'], ['value__autocorrelation__lag_5', '0.464639525765'], ['value__autocorrelation__lag_4', '0.535012070945'], ['value__autocorrelation__lag_3', '0.411575406169'], ['value__autocorrelation__lag_2', '0.367658131978'], ['value__autocorrelation__lag_1', '0.51547994425'], ['value__augmented_dickey_fuller', '-0.804122034203'], ['value__ar_coefficient__k_10__coeff_4', '0.283685319392'], ['value__ar_coefficient__k_10__coeff_3', '-0.066542370683'], ['value__ar_coefficient__k_10__coeff_2', '-0.0432470001474'], ['value__ar_coefficient__k_10__coeff_1', '0.163578948116'], ['value__ar_coefficient__k_10__coeff_0', '904.439185079'], ['value__approximate_entropy__m_2__r_0.9', '0.679727853081'], ['value__approximate_entropy__m_2__r_0.7', '0.770566235381'], ['value__approximate_entropy__m_2__r_0.5', '0.816629919728'], ['value__approximate_entropy__m_2__r_0.3', '0.556059428563'], ['value__approximate_entropy__m_2__r_0.1', '0.0768916250217'], ['value__absolute_sum_of_changes', '2755.0'], ['value__abs_energy', '5983503421.0']]

tests/baseline/tsfresh_features_test.py:355: AssertionError
```